modules/core/src/main/java/org/apache/ignite/internal/IgniteFeatures.java

```diff
@@ -64,8 +64,11 @@
     /** Supports tracking update counter for transactions. */
     TX_TRACKING_UPDATE_COUNTER(12),
 
-    /** Support new security processor */
-    IGNITE_SECURITY_PROCESSOR(13);
+    /** Support new security processor. */
+    IGNITE_SECURITY_PROCESSOR(13),
+
+    /** Indexing enabled. */
+    INDEXING(14);
```

Comment:
code should be reserved on https://ggsystems.atlassian.net/wiki/spaces/GG/pages/1192198276/Community+edition+features+list

Reply:
Code reserved.